lib/parsers/livesplit.rb

```diff
@@ -85,6 +85,7 @@ def v1_6(xml, fast)
         raise "Not a proper LiveSplit run file"
       end
 
+      run[:history].unshift(run[:time]) unless fast || run[:time].in?(run[:history])
```

Comment:
Ah, that makes a lot of sense.  I'll remove these tonight.  I didn't think about the implications of other runs missing and agree that it would do more harm then good.  Do you know if LiveSplit has always tracked split history?  If it did would adding all completed runs to the history by going through the split ID's be a better solution?